scripts/check_frontend_coverage.py

```diff
@@ -232,6 +232,7 @@
     'player-correctness-feedback-enabled.service.ts',
     'player-transcript.service.ts',
     'Polyfills.ts',
+    'positive-int-editor.directive.ts',
```

Comment:
Can we completely cover the logic in the file please? Lets not add to the list

Reply:
Done -- added a spec file